build/Makefile

```diff
@@ -45,14 +45,38 @@
 # you will need to figure out what the equivalent flags are
 # and may need to update this section
 ifeq "$(FC)" "gfortran"
- FLAGS_NOAH = -p -g -ffree-form -fdefault-real-8 -ffree-line-length-none -fmax-errors=0 -fbacktrace -Wno-unused -Wno-unused-dummy-argument
- FLAGS_COMM = -p -g -Wall -ffree-line-length-none -fmax-errors=0 -fbacktrace -fcheck=bounds
- FLAGS_SUMMA = -p -g -Wall -ffree-line-length-none -fmax-errors=0 -fbacktrace -fcheck=bounds
+
+ # Define the flags for openMP
+ FLAGS_OMP =
+ #FLAGS_OMP = -fopenmp
+
+ # Production run
```

Comment:
Here (and for ifort and debug) can you set the FLAGS_NOAH to just be the flags needed for Noah, then have FLAGS_COMM be common(?) flags (O3,free, etc) and FLAGS_SUMMA be flags only used by summa (and not Noah?)